cartridge/roles.lua

```diff
@@ -299,8 +299,22 @@ local function validate_config(conf_new, conf_old)
         error(err, 2)
     end
 
+    local disabled_roles = {}
```

Comment:
Please, add a comment why do you check "disabled" roles, not just "enabled".

Is it related anyhow to permanent/hidden roles? Validate config should not be skipped for them.